src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -776,7 +776,7 @@ var TorrentsTable = new Class({
             // state_icon
             this.columns['state_icon'].updateTd = function (td, row) {
                 var state = this.getRowValue(row);
-
+                // normalize states
```

Comment:
I'd like to see some explanation in comment here to avoid some further confusion (to the developers no longer had any question why there was no `default` case).

Reply:
or add an explicit: `default: break;`

Reply:
I've opted for the latter approach, and squashed the changes.